src/SendGrid/Helpers/Mail/Validators.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using SendGrid.Helpers.Mail.Model;
+
+namespace SendGrid.Helpers.Mail
+{
+	/// <summary>
+	/// This class is an extension method to check if the parameters is null or empty
+	/// </summary>
+	public static class Validators
```

Comment:
I think this class should be named `EmailAddressExtensions` to better convey what it contains.

Reply:
Sure i will change this